client/components/web-scraping/FlightCard.tsx

```diff
@@ -0,0 +1,108 @@
+  const arrival = new Date(flight.arrivalTime);
+  const duration = arrival.getTime() - departure.getTime();
+  const timeOptions: Intl.DateTimeFormatOptions = { hour: '2-digit', minute: '2-digit' };
+  const dateOptions: Intl.DateTimeFormatOptions = { month: 'short', day: 'numeric' };
+
+  return (
+    <Card style={{ marginTop: '20px' }} variant={'outlined'} data-test={FLIGHT_TAG.card}>
+      <CardContent>
+        <div className={styles.container}>
+          <div className={styles.origin} data-test={FLIGHT_TAG.origin}>
+            <Typography fontSize={'small'} className={styles.date}>
+              {departure.toLocaleDateString('en-US', dateOptions)}
+            </Typography>
+            <Typography className={styles.time} data-test={FLIGHT_TAG.time}>
+              {departure.toLocaleTimeString('en-US', timeOptions)}
+            </Typography>
+            <Typography className={styles.city}>{flight.fromCity}</Typography>
+            <Typography
+              className={styles.airportCode}
+              fontSize="small"
+              data-test={FLIGHT_TAG.airportCode}
+            >{`(${flight.fromCode})`}</Typography>
+          </div>
+          <div className={styles.middle}>
+            <Typography fontSize={'small'} className={styles.duration}>
+              {formatTime(duration)}
+            </Typography>
+            <div className={styles.transition}>
+              <div className={styles.circleContainer}>
+                <CircleIcon className={styles.circle} />
+              </div>
+              <div className={styles.line}></div>
+              <div className={styles.arrow}>
+                <FlightIcon fontSize="small" />
+              </div>
+              <div className={styles.line}></div>
+              <div className={styles.circleContainer}>
+                <CircleIcon className={styles.circle} />
+              </div>
+            </div>
+            <Typography fontSize={'small'} className={styles.airline} data-test={FLIGHT_TAG.airline}>
+              {flight.airline}
+            </Typography>
+          </div>
+          <div className={styles.destination} data-test={FLIGHT_TAG.destination}>
+            <Typography fontSize="small" className={styles.date}>
+              {arrival.toLocaleDateString('en-US', dateOptions)}
+            </Typography>
+            <Typography className={styles.time} data-test={FLIGHT_TAG.time}>
+              {arrival.toLocaleTimeString('en-US', timeOptions)}
+            </Typography>
+            <Typography className={styles.city}>{flight.toCity}</Typography>
+            <Typography
+              className={styles.airportCode}
+              data-test={FLIGHT_TAG.airportCode}
+              fontSize="small"
+            >{`(${flight.toCode})`}</Typography>
+          </div>
+        </div>
+      </CardContent>
+      <CardActions
+        sx={(theme) => ({
+          backgroundColor: theme.palette.mode === 'dark' ? '#171717' : '#fafafa',
```

Comment:
If possible, please try to use colors from the theme in order to keep our UI consistent.
If you need to introduce new colors, let's add them to the theme and use them from it :)

https://mui.com/material-ui/customization/palette/#adding-new-colors

Reply:
Good point, fixed!